pyperformance/_manifest.py

```diff
@@ -275,8 +275,7 @@ def _iter_sections(lines):
 
 
 def _parse_manifest_file(filename):
-    relroot = os.path.dirname(filename)
-    filename = _utils.resolve_file(filename, relroot)
+    filename = os.path.abspath(filename)
```

Comment:
I'd say drop both lines here and add `filename = _utils.resolve_file(filename)` up in `load_manifest()`.